networking/ipv6.md

```diff
@@ -1,145 +1,134 @@
+  - The sysctl setting, `net.ipv6.conf.all.forwarding`, is set to `1`.    
+    This ensures both Kubernetes service traffic and {{site.prodname}} traffic is forwarded appropriately.
+  - A default IPv6 route
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
-| `--cluster-cidr` | Should be set to match the {{site.prodname}} IPv6 IPPool. |
+### How to
 
-##### kube-scheduler
+>**Note**: The following tasks work only for new clusters; you cannot migrate existing IPv4 clusters to IPv6.
+{: .alert .alert-info}
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
+**Manifest install**
+- [Enable dual stack, Kubernetes](#enable-dual-stack-kubernetes)
+- [Enable IPv6-only, initial install, Kubernetes](#enable-ipv6-only-initial-install-kubernetes)
+- [Enable IPv6-only, after install, Kubernetes](#enable-ipv6-only-after-install-kubernetes)
 
-##### kubelet
+**OpenShift or Quickstart operator install**
+- [Enable dual stack, OpenShift](#enable-dual-stack-openshift)
+- [Enable IPv6-only, OpenShift](#enable-ipv6-only-openshift)
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--address` | Should be set to the appropriate IPv6 address or `::` for all IPv6 addresses. |
-| `--cluster-dns` | Should be set to the IPv6 address that will be used for the service DNS, this must be in the range used for `--service-cluster-ip-range`. |
-| `--node-ip` | Should be set to the IPv6 address of the node. |
+**Optional**
+- [Configure the Kubernetes control plane to operate over IPv6](#configure-the-kubernetes-control-plane-to-operate-over-ipv6)
 
-##### kube-proxy
+#### Enable dual stack, Kubernetes
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--bind-address` | Should be set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
-| `--cluster-cidr` | Should be set to match the {{site.prodname}} IPv6 IPPool. |
+1. Set up a new cluster following the Kubernetes {% include open-new-window.html text='prerequisites' url='https://kubernetes.io/docs/concepts/services-networking/dual-stack/#prerequisites' %} and {% include open-new-window.html text='enablement steps' url='https://kubernetes.io/docs/concepts/services-networking/dual-stack/#enable-ipv4-ipv6-dual-stack' %}.
 
-### Enabling IPv6 support in {{site.prodname}}
+1. Using the [{{site.prodname}} Kubernetes install guide]({{site.baseurl}}/getting-started/kubernetes/self-managed-onprem/onpremises), download the right {{site.prodname}} manifest for the cluster, and datastore type. 
 
-To enable IPv6 support when installing {{site.prodname}} follow the
-steps below.
+1. Edit the CNI config (`calico-config` ConfigMap in the manifest), and enable IP address allocation by setting both modes to true.
```

Comment:
"... enable IPv4 and IPv6 address allocation by ..."

Reply:
fixed.